### PR TITLE
Draft: Logistic regression in python

### LINKED ISSUE #247: Logistic Regression in Python

### PR DESCRIPTION
**Pull request by orladoylenvs**
Closes #247

### CODE REVIEW

File: python/logistic_regression.qmd [hunk 1]
```diff
@@ -0,0 +1,135 @@
...
+
+#modelling
+import statsmodels.api as sm
+from sklearn.linear_model import LogisticRegression
+```
+
+
+# Background
+
+Logistic regression models the relationship between a binary variables and set of explanatory variables. The logit of expectation is explained as linear for of explanatory variables. If we observed $(y_i, x_i),$ where $y_i$ is a Bernoulli variable and $x_i$ a vector of explanatory variables, the model for $\pi_i = P(y_i=1)$ is
```
Comment (thisisnic) [inline python/logistic_regression.qmd:20]:
```suggestion
Logistic regression models the relationship between a binary variable and set of explanatory variables. The logit of expectation is explained as linear for of explanatory variables. If we observed $(y_i, x_i),$ where $y_i$ is a Bernoulli variable and $x_i$ a vector of explanatory variables, the model for $\pi_i = P(y_i=1)$ is
```

Comment (thisisnic) [reply python/logistic_regression.qmd:20]: Also I think there may be a typo in "for of"

File: python/logistic_regression.qmd [hunk 2]
```diff
@@ -0,0 +1,135 @@
...
+
+#create weight loss factor while respecting missing values
+# 1: patients with a weight loss of more than zero
+# 0: patients a weight loss of zero oe less
+lung2["wt_grp"] = np.where(lung2["wt.loss"].isnull(), np.nan, (lung2["wt.loss"] > 0).astype(int))
+```
+
+# Logistic Regression Modelling
+
+Let's further prepare our data for modelling by selecting the explanatory variables and the dependent variable. The python packages that we are are aware of require complete (i.e. no missing values) data so for convenience of demonstrating these methods we will drop rows with missing values.
```
Comment (thisisnic) [inline python/logistic_regression.qmd:45]:
```suggestion
Let's further prepare our data for modelling by selecting the explanatory variables and the dependent variable. The Python packages that we are are aware of require complete (i.e. no missing values) data so for convenience of demonstrating these methods we will drop rows with missing values. 
```

File: python/logistic_regression.qmd [hunk 3]
```diff
@@ -0,0 +1,135 @@
...
+#intercept column
+x_sm = sm.add_constant(x)
+
+#fit model
+lr_sm = sm.Logit(y, x_sm).fit() 
+print(lr_sm.summary())
+```
+
+### Model fitting
+In addition to the information contained in the summary, we can display the model coefficients as odds raios:
```
Comment (thisisnic) [inline python/logistic_regression.qmd:72]:
```suggestion
In addition to the information contained in the summary, we can display the model coefficients as odds ratios:
```

**Review by thisisnic — COMMENTED** [review]
Looks great, thanks for adding the dataset!  Just a few tiny typos from me to update!

**Comment by orladoylenvs** [comment]
Thanks @thisisnic 
@statasaurus - ready for your review when you get a chance!

**Review by statasaurus — APPROVED** [review]
🦄 Great job Thank you!!
